src/shared/services/document-management/document-management.service.ts

```diff
@@ -55,16 +63,37 @@ export class DocumentManagementService {
     }
     if (this.isImage(fileExtension)) {
       return DocumentManagementService.IMAGE;
+    } else if (this.isWord(fileExtension)) {
+      return DocumentManagementService.WORD;
+    } else if (this.isExcel(fileExtension)) {
+      return DocumentManagementService.EXCEL;
+    } else if (this.isPowerpoint(fileExtension)) {
+      return DocumentManagementService.POWERPOINT;
+    } else if (fileExtension.toLowerCase() === 'txt') {
+      return DocumentManagementService.TXT;
+    } else if (fileExtension.toLowerCase() === 'rtf') {
+      return DocumentManagementService.RTF;
     } else if (fileExtension.toLowerCase() === 'pdf') {
       return DocumentManagementService.PDF;
     } else {
-      console.warn(`Unknown content type with the file extension: ${fileExtension}`);
       return fileExtension;
     }
   }
 
   isImage(imageType: string) {
```

Comment:
_Nitpick:_
Should have a return type of `boolean` and a `public`/`private` indicator. The same for all of the `isX` ones in here, actually...

Reply:
You are right of course.